src/Install.php

```diff
@@ -463,6 +463,18 @@ protected static function delete_obsolete_notes() {
 			'wc-admin-store-notice-giving-feedback',
 		);
 
+		$additional_obsolete_notes_names = apply_filters(
+			'woocommerce_admin_obsolete_notes_names',
```

Comment:
Not a necessity, but we could directly filter the notes from above so we don't need to merge multiple arrays.  I generally favor providing more control than less.

Reply:
By doing it this way, we ensure that no other code can override our removing of these notes, which we definitely want gone. If any filter handler didn't merge the passed in array with whatever they wanted removed, our notes would not be removed. Of course, this same basic problem would exist with any other filter handler wanted removed. It's a general brittleness of this approach.

We could revisit this in a future PR if wanted.